habitat/plan.sh

```diff
@@ -1,68 +1,76 @@
 pkg_name=inspec
 pkg_origin=chef
-pkg_version=$(cat "VERSION")
 pkg_description="InSpec is an open-source testing framework for infrastructure
   with a human- and machine-readable language for specifying compliance,
   security and policy requirements."
-pkg_upstream_url=https://github.com/chef/inspec
+pkg_upstream_url=https://www.inspec.io/
 pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"
 pkg_license=('Apache-2.0')
-pkg_source=false
 pkg_deps=(
+  core/busybox-static
+  core/cacerts
   core/coreutils
   core/libxml2
   core/libxslt
-  core/ruby
   core/net-tools
-  core/busybox-static
+  core/ruby
 )
 pkg_build_deps=(
   core/bundler
-  core/coreutils
   core/gcc
   core/make
   core/readline
+  core/sed
 )
 pkg_bin_dirs=(bin)
 
-do_download() {
-  return 0
+pkg_version() {
+  cat VERSION
 }
 
-do_verify() {
-  return 0
-}
-
-do_unpack() {
-  return 0
+do_before() {
+  if [[ ! -f VERSION ]]; then
+    exit_with "habitat/VERSION not found. Run 'rake release_habitat' to ensure it exists" 1
+  fi
+  update_pkg_version
 }
 
 do_prepare() {
   # Create a Gemfile with what we need
-  cat > Gemfile <<GEMFILE
+  cat > "$CACHE_PATH/Gemfile" <<GEMFILE
```

Comment:
@elliott-davis the main reason for this is that there is now way to bundle install without having bundler write out a .bundle/config to the same directory where the Gemfile is located, so if you build using the "real" Gemfile, then you won't be able to bundle anything on your local machine, because bundler will save those settings.

I'm pushing up a commit where we use `path: $SRC_PATH` instead of `= $pkg_version`. This will make it use the current version on disk rather than pulling from rubygems.